cmd/port-layer-server/main.go

```diff
@@ -76,8 +79,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	// load the vch config
+	src, err := extraconfig.GuestInfoSource()
+	if err != nil {
+		log.Errorf("Unable to load configuration from guestinfo")
+		return
+	}
+
+	extraconfig.Decode(src, &vchConfig)
+
 	logcfg := viclog.NewLoggingConfig()
-	if ploptions.PortLayerOptions.Debug {
+	if vchConfig.Diagnostics.DebugLevel {
```

Comment:
The original `ploptions.PortlayerOptions.Debug` has default value of `debug` and is not initialized using `vchConfig.Diagnositics.DebugLevel`. Based on discussion with Hasan, I find that there is an existing issue #595 which requests obtaining all the portlayeroptions via extraconfig (what I did here).